Userland/Games/FlappyBug/Game.h

```diff
@@ -28,7 +83,7 @@ class Game final : public GUI::Frame {
     Function<u32(u32)> on_game_end;
 
 private:
-    Game();
+    Game(Bug&&);
```

Comment:
The `&&` is not really necessary as the constructor definition moves the `Bug` argument.